packages/blockchain-utils-validation/package.json

```diff
@@ -33,6 +33,7 @@
     "@tendermint/sig": "^0.6.0",
     "@zondax/filecoin-signing-tools": "^0.13.0",
     "caip": "^0.9.2",
+    "near-api-js": "^0.36.3",
```

Comment:
Looks like this dep is only used for tests, can you move it to devDeps?

Reply:
yep, will do